### DISCUSSION TITLE
flixel bug

### DISCUSSION THREAD

**Discussion by aless26pic** [HaxeFlixel/flixel#2866] [Q&A]
C:/HaxeToolkit/haxe/lib/flixel/5,3,1/flixel/animation/FlxAnimation.hx:8: characters 44-45 : Expected extends or implements or {

**Comment by Geokureli** [comment]
@aless26pic Converted to a discussion because the line you linked is perfectly fine.
https://github.com/HaxeFlixel/flixel/blob/master/flixel/animation/FlxAnimation.hx#L8

The error must be on your end. Is this the only error you're getting, and do you get it when you start a new project?

**Reply by Snirozu** [reply]
![](https://github.com/HaxeFlixel/flixel/assets/72814880/d28defeb-6cec-4007-bc93-2263aff0e3bd)
uh

**Reply by Geokureli** [reply]
oops, fixed
